Modules/Directory.Build.props

```
@@ -12,4 +12,21 @@
             CS8777,CS8819,CS8824,CS8825,CS8847
         </WarningsAsErrors>
     </PropertyGroup>
+
+    <ItemGroup Condition="'$(Configuration)' != 'Release' And $(MSBuildProjectFile.Contains('Relationship'))">
```

Btw, shouldn't it be called "Relationship**s**"?
```suggestion
    <ItemGroup Condition="'$(Configuration)' != 'Release' And $(MSBuildProjectFile.Contains('Relationships'))">
```